.changeset/strong-lobsters-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/store": patch
```

Comment:
need to bump store/world protocol versions

Reply:
done